src/uct/Makefile.am

```diff
@@ -34,7 +35,12 @@ noinst_HEADERS = \
 	sm/mm/base/mm_def.h \
 	sm/mm/base/mm_md.h \
 	sm/self/self.h \
-	tcp/tcp.h
+	tcp/tcp.h \
+	tcp/sockcm/sockcm_def.h \
```

Comment:
FYI: I'd suggest to move TCP transport files under subdir, e.g tcp/tl (in addition to tcp/sockcm). @dmitrygladkov can do it after this PR is merged.

Reply:
Noted.

Reply:
> @dmitrygladkov can do it after this PR is merged.

sure